maintenance_plan/hooks.py

```diff
@@ -23,17 +21,17 @@ def post_init_hook(cr, registry):
 
         for equipment in equipments:
             request = equipment.maintenance_ids.filtered(
-                lambda r: r.maintenance_type == "preventive"
+                lambda r, equipment=equipment: r.maintenance_type == "preventive"
```

Comment:
Why this change?

Reply:
B023 Function definition does not bind loop variable `equipment

<https://docs.astral.sh/ruff/rules/function-uses-loop-variable/>`

Reply:
I see, it is a new requirement from ruff :+1: